frontend/src/app/components/enterprise-banner/enterprise-banner.component.ts

```diff
@@ -6,7 +6,7 @@ import {I18nService} from "core-app/modules/common/i18n/i18n.service";
   selector: 'enterprise-banner',
   styleUrls: ['./enterprise-banner.component.sass'],
   template: `
-      <div class="notification-box -ee-upsale"
+      <div class="op-enterprise-banner-notification-box -ee-upsale"
            [ngClass]="{'-left-margin': leftMargin }">
         <div class="notification-box--content">
```

Comment:
This should be an element of the block. Also, since the whole component is called `enterprise-banner`, maybe just call the block `enterprise-banner`, though I'm not sure what it's exact use is, so I'll leave that up to you.